aws/eks-helm/metrics-server/variables.tf

```diff
@@ -0,0 +1,16 @@
+variable "namespace" {
+  description = "The Namespace of the services."
+  type        = string
+  default     = "kube-system"
+}
+variable "helm_release_name" {
+  description = "The Helm release of the services."
+  type        = string
+  default     = "metrics-server"
+}
+
+variable "helm_chart_version" {
+  default     = "3.11.0"
```

Comment:
use the latest version https://artifacthub.io/packages/helm/metrics-server/metrics-server

Reply:
updated